classes/author.rb

```diff
@@ -0,0 +1,16 @@
+class Author
```

Comment:
- Optional
Author data is not being saved in the local storage (JSON file). Kindly fix this error. Since, the previous reviewer didn't raise this as required, I am marking this as optional.